src/Microsoft.DotNet.Wpf/src/PresentationBuildTasks/Microsoft.WinFX.targets

```diff
@@ -377,18 +380,22 @@
   <PropertyGroup>
 
        <MarkupCompilePass2ForMainAssemblyDependsOn>
+                     CreateTemporaryTargetAssemblyProject;
                      GenerateTemporaryTargetAssembly;
+                     GenerateTemporaryTargetAssembly2;
                      MarkupCompilePass2;
                      AfterMarkupCompilePass2;
                      CleanupTemporaryTargetAssembly
        </MarkupCompilePass2ForMainAssemblyDependsOn>
 
-       <_CompileTargetNameForLocalType Condition="'$(_CompileTargetNameForLocalType)' == ''">_CompileTemporaryAssembly</_CompileTargetNameForLocalType>
+        <_CompileTargetNameForLocalType Condition="'$(_CompileTargetNameForLocalType)' == ''">_CompileTemporaryAssembly</_CompileTargetNameForLocalType>
 
+        <_ResolveProjectReferencesTargetName Condition="'$(IncludePackageReferencesDuringMarkupCompilation)' == 'true'">ResolveProjectReferences</_ResolveProjectReferencesTargetName>
```

Comment:
I should have caught this the first time around... 

For public properties, strongly consider using negative tests like `"'$(IncludePackageReferencesDuringMarkupCompilation)' != 'false'"`

Inevitably, these properties have a tendency to be used somewhere else besides the props/targets file that initially declared it. Then you'll have to *initialize* it everywhere, which tends to be another common problem. 

The safest pattern for public properties, if you can get away with it is...

- No initialization
- Negative tests ( `!= false`) 

You don't really have to drop the initialization in this file if you feel really attached to it (it's somewhat benign, and has a mildly self-documenting value), but I don't see a strong value there.